src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -1100,44 +1094,6 @@ def do_revert(self, args):
         else:
             gLogger.notice("Software reverted to", result["Value"])
 
-    def do_add(self, args):
```

Comment:
Why did you remove this entirely?

Reply:
This call is to add a system instance. The system instances elimination is the whole purpose of this PR. This call is no more relevant